<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>abbClass</class>
 <widget class="QMainWindow" name="abbClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>abb</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="author">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1301</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Chessboard">
     <attribute name="title">
      <string>Chessboard</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="VisionSystem">
     <attribute name="title">
      <string>Vision System</string>
     </attribute>
     <widget class="QPushButton" name="btnPauseOrResume">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>480</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblProcessed">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>450</width>
        <height>450</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblOriginal">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>450</width>
        <height>450</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="txtXYRadius">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>460</y>
        <width>551</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="zmienne">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>40</y>
        <width>121</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Author</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>321</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Author:&lt;br/&gt;Michał Sokołowski&lt;/p&gt;&lt;p&gt;Praca na inżynierkę&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Kielce University of Technology, 2019&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="abb.qrc"/>
 </resources>
 <connections/>
</ui>
